bgpd/bgp_routemap.c

```diff
@@ -5501,6 +5579,51 @@ DEFUN_YANG (no_match_ecommunity,
 }
 
 
+DEFPY_YANG (set_ecommunity_delete,
+	    set_ecommunity_delete_cmd,
+            "set extended-comm-list <(1-99)|(100-500)|EXTCOMMUNITY_LIST_NAME> delete",
+	    SET_STR
+	    "set BGP extended community list (for deletion)\n"
+	    "Extended community-list number (standard)\n"
+	    "Extended community-list number (expanded)\n"
+	    "Extended community-list name\n"
+            "Delete matching extended communities\n")
+{
+	const char *xpath =
+		"./set-action[action='frr-bgp-route-map:extended-comm-list-delete']";
+	char xpath_value[XPATH_MAXLEN];
+	int idx_comm_list = 2;
```

Comment:
We can tag `(1-99)|(100-500)|EXTCOMMUNITY_LIST_NAME$list>` and use it instead of using an old method (DEFUN-way).

Reply:
Come again please!

Reply:
m? :)

Reply:
I don't understand your demand. can you reformulate please.